app/models/throttle.rb

```diff
@@ -18,7 +18,7 @@ class Throttle < ApplicationRecord
   THROTTLE_CONFIG = {
     idv_acuant: {
       max_attempts: IdentityConfig.store.acuant_max_attempts,
```

Comment:
Similar to what @aduth said for `acuant_max_attempts`. If we don't want to do it right now we should probably write a ticket to generalize configs and code away from acuant to doc_auth where applicable.

Reply:
done in 39eae3465